internal/v2/remote_signer/remote_signer.go

```diff
@@ -2,13 +2,77 @@
 package remotesigner
 
 import (
+	"io"
 	"crypto"
-	"crypto/tls"
+	"crypto/x509"
+	s2av2pb "github.com/google/s2a-go/internal/proto/v2/s2a_go_proto"
+	commonpbv1 "github.com/google/s2a-go/internal/proto/common_go_proto"
 )
 
+type RemoteSigner struct {
+	leafCert *x509.Certificate
+	cstream s2av2pb.S2AService_SetUpSessionClient
+	hostname string
+}
+
+func GetCert(s RemoteSigner) *x509.Certificate {
+	return s.leafCert
+}
+
+func GetStream(s RemoteSigner) s2av2pb.S2AService_SetUpSessionClient {
+	return s.cstream
+}
+
+func GetHostname(s RemoteSigner) string {
```

Comment:
Should this function be unexported? If so, can we move it to the bottom of the file.